src/Providers/StringMacroServiceProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ARKEcosystem\Foundation\Providers;
+
+use ARKEcosystem\Foundation\Support\UrlInStringWrapper;
+use Illuminate\Support\ServiceProvider;
+use Illuminate\Support\Str;
+
+class StringMacroServiceProvider extends ServiceProvider
+{
+    public function register(): void
```

Comment:
This method should be safe to remove because it has been optional for a few major versions of Laravel.